Cargo.toml

```
@@ -90,3 +90,4 @@ tracing = "0.1"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter", "json"] }
 turmoil = "0.6.4"
 vbs = "0.1"
+zeroize = { version = "^1.8", features = ["zeroize_derive"] }
```

```suggestion
zeroize = { version = "1.8", features = ["zeroize_derive"] }
```
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements